Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -244,12 +242,18 @@ extension PathHierarchy {
                     }
                 } catch DisambiguationContainer.Error.lookupCollision(let collisions) {
                     func handleWrappedCollision() throws -> Node {
-                        let match = try handleCollision(node: node, remaining: remaining, collisions: collisions, onlyFindSymbols: onlyFindSymbols, rawPathForError: rawPathForError)
```

Comment:
FYI: This is an example of what I mentioned in the meeting where I intentionally put all the arguments on the same line because they're all named the same as the parameter. 

Like I said; I do sometimes as a hint to the reader that there's nothing interesting going on on this line... after skimming the first 1 or 2 arguments, the rest of the line is what the reader would expect.

Reply:
You can keep doing this and preserve the formatting as described in https://github.com/swiftlang/swift-format/blob/main/Documentation/IgnoringSource.md

Reply:
I personally think that is a small price to pay for having automated formatting rules. You can disable it for this line by using https://github.com/swiftlang/swift-format/blob/main/Documentation/IgnoringSource.md

Reply:
Yes, definitely. I was just saying that I sometimes intentionally format my code like this.